.github/dependabot.yml

```diff
@@ -5,13 +5,14 @@ updates:
     schedule:
       interval: weekly
     reviewers:
-      - amimart
       - ccamel
       - fredericvilcot
       - ErikssonJoakim
+      - lolottetheclash
```

Comment:
I will have added the whole team as reviewers to keep the consistency about updates knowledge 😌